<launch>

	<node pkg="roomba_500driver_meiji" name="roomba_driver" type="main500"/>

	<node pkg="hokuyo_node" name="hokuyo_node" type="hokuyo_node">
		<param name="port" value="/dev/ttyACM0"/>
	</node>

	<node pkg="roomba_teleop_meiji" name="electric_joystick_drive" type="electric_joystick_drive"/>

	<node pkg="joy" name="joy_node" type="joy_node">
		<param name="dev" value="/dev/input/js0"/>
	</node>

	<arg name="map_file" default="$(find chibi19_b)/mapdata/DMapData.yaml"/>
	<node pkg="map_server" name="map_server" type="map_server" args="$(arg map_file)">
	</node>

	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find chibi19_b)/config/rviz/Localization.rviz"/>

	<node pkg="chibi19_b" type="Localizer" name="Localizer" output="screen">
		<rosparam file="$(find chibi19_b)/config/param/Localizer.yaml" command="load"/>
	</node>

</launch>
